### ISSUE TITLE
DEB package contains /.git dir and other source files

### ISSUE THREAD

**Issue by SheliakLyr** [ryanmjacobs/darkhttpd#1]
Files in deb package:

```
dpkg-deb --contents darkhttpd.deb | tr -s ' ' | cut -d' ' -f 6
./
./usr/
./usr/bin/
./usr/bin/darkhttpd
./Makefile
./darkhttpd.c
./darkhttpd
./.gitignore
./README.md
./.git/
./.git/description
./.git/refs/
./.git/refs/remotes/
./.git/refs/remotes/origin/
./.git/refs/remotes/origin/HEAD
./.git/refs/heads/
./.git/refs/heads/master
./.git/refs/tags/
./.git/objects/
./.git/objects/pack/
./.git/objects/pack/pack-7822fc78e375aa535f362e22ef50badbf2186e43.idx
./.git/objects/pack/pack-7822fc78e375aa535f362e22ef50badbf2186e43.pack
./.git/objects/info/
./.git/branches/
./.git/HEAD
./.git/logs/
./.git/logs/refs/
./.git/logs/refs/remotes/
./.git/logs/refs/remotes/origin/
./.git/logs/refs/remotes/origin/HEAD
./.git/logs/refs/heads/
./.git/logs/refs/heads/master
./.git/logs/HEAD
./.git/hooks/
./.git/hooks/pre-commit.sample
./.git/hooks/applypatch-msg.sample
./.git/hooks/pre-applypatch.sample
./.git/hooks/update.sample
./.git/hooks/pre-push.sample
./.git/hooks/prepare-commit-msg.sample
./.git/hooks/post-update.sample
./.git/hooks/pre-rebase.sample
./.git/hooks/commit-msg.sample
./.git/packed-refs
./.git/ORIG_HEAD
./.git/config
./.git/FETCH_HEAD
./.git/index
./.git/info/
./.git/info/exclude
```

Imagine my surprise after I've found I have a git repo in root (/)...

**Comment by ryanmjacobs** [comment]
Yikes that's not good. I'll get around to that on Monday.

**Comment by ryanmjacobs** [comment]
I think I might have fixed the issue with 2c2d444, but I don't have a debian-based system to test it on.
